app/src/main/java/org/p2p/wallet/common/mvp/BaseMvpBottomSheet.kt

```diff
@@ -42,6 +44,12 @@ abstract class BaseMvpBottomSheet<V : MvpView, P : MvpPresenter<V>>(
         presenter.detach()
     }
 
+    protected fun expandToFitAllContent() {
```

Comment:
private

Reply:
we could override it with `= Unit` to prevent expanding in future

Reply:
sounds like a boolean flag for me, i wouldn't ever think that i need to override method to Unit if i want to collapse bottom sheet..